examples/http/sonicwall_failover_load_balancing_groups.js

```diff
@@ -0,0 +1,158 @@
+/**
+ * Domotz Custom Driver 
+ * Name: Sonicwall - Failover Load Balancing Groups
+ * Description: Monitors Sonicwall failover load balancing groups. 
+ * 
+ * Communication protocol is HTTPS
+ * 
+ * Tested on SonicWALL TZ370W
+ *
+ * Creates a Custom Driver Table with the following columns:
+ *      - Group Name: failover group name
+ *      - Member Name: failover group member name
+ *      - IP Address
+ *      - Link Status
+ *      - Load balancing status
+ *      - Probe status
+ *      - Main target status
+ *      - Alternate target status
+ **/
+
+var sonicWallAPIPort = 443;
```

Comment:
If the port is always 443, it might be better to remove this variable, as the default port for HTTPS is 443 when the protocol is set to HTTPS. 
However, if the port could vary, it's better to use a parameter.